cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/includes/top.jsp

```diff
@@ -28,32 +28,47 @@
 <%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %>
 
 <html lang="en">
-<head>
-  <meta charset="UTF-8" />
-
-  <%-- <title>CAS &#8211; Central Authentication Service</title> --%>
-  <title>Open Science Framework | Sign In</title>
-
-  <spring:theme code="standard.custom.css.file" var="customCssFile" />
-  <link rel="stylesheet" href="<c:url value="${customCssFile}" />" />
-  <link rel="icon" href="<c:url value="/favicon.ico" />" type="image/x-icon" />
-
-  <!--[if lt IE 9]>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
-  <![endif]-->
-
-  <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,300,700' rel='stylesheet' type='text/css'>
-</head>
-<body id="cas">
-    <div id="container">
-        </br>
-        <header>
-            <spring:eval var="osfUrl" expression="@casProperties.getProperty('osf.url')" />
-            <a id="logo" href="${osfUrl}" title="<spring:message code="logo.title" />">Open Science Framework | Sign In</a>
+    <head>
+        <meta charset="UTF-8" />
+
+        <%-- <title>CAS &#8211; Central Authentication Service</title> --%>
+        <title>Open Science Framework | Sign In </title>
+
+        <spring:theme code="standard.custom.css.file" var="customCssFile" />
+        <link rel="stylesheet" href="<c:url value="${customCssFile}" />" />
+        <link rel="icon" href="<c:url value="/favicon.ico" />" type="image/x-icon" />
+
+        <!--[if lt IE 9]>
+            <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
+        <![endif]-->
+
+        <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,300,700' rel='stylesheet' type='text/css'>
+    </head>
+
+    <body id="cas" onload="selectFocus()">
+        <div id="container">
+            <br>
+            <header>
+                <div class="center">
+                    <spring:eval var="osfUrl" expression="@casProperties.getProperty('osf.url')" />
+                    <a id="logo" class="center" href="${osfUrl}" title="<spring:message code="logo.title" />">Open Science Framework | Sign In</a>
+                </div>
+                <br>
+                <div class="center">
+                    <span id="title">
+                        <span class="title-full">Open&nbsp;Science&nbsp;Framework</span>
+                        <span class="title-abbr">Open&nbsp;Science<br>Framework</span>
+                        <c:if test="${campaign eq 'INSTITUTION'}"><span> for&nbsp;Institutions</span></c:if>
+                    </span>
+                </div>
+                <div class="responsive">
+                    <c:if test="${campaign eq 'INSTITUTION'}">
```

Comment:
Agreed. Leave it as it is now. Add this change for https://github.com/CenterForOpenScience/cas-overlay/pull/24.